driver/gpio/imxgpio/imxgpio.cpp

```diff
@@ -45,9 +46,13 @@ namespace {
             case IMX_GPIO_PULL_DEFAULT:
                 return "PullDefault";
             case IMX8M_GPIO_PULL_UP:
+            case IMX8MM_GPIO_PULL_UP:
                 return "PullUp";
+            case IMX8MM_GPIO_PULL_DOWN:
+                return "PullDown";
             case IMX_GPIO_PULL_DISABLE:
                 return "PullNone";
+            case IMX_GPIO_PULL_INVALID:
```

Comment:
IMX_GPIO_PULL_INVALID should only be found in the pullUp or pulldown static class members when pull up or pull down isn't supported by the hw.  It should not found natively in the HW, nor should it be generated by GpioPullModeToImxPullMode.